src/tests/multihost/alltests/conftest.py

```diff
@@ -1057,6 +1057,59 @@ def remove_journalsssd():
     request.addfinalizer(remove_journalsssd)
 
 
+@pytest.fixture(scope='class')
+def sssdproxyldap_test(session_multihost, request):
+    """ Configure  sssdproxyldap
+        Configure  sssd.conf
+        Configure  nslcd.conf
+        Transport sssdproxyldap.sh to client machine
+        configure password for ldap user
+    """
+    tools = sssdTools(session_multihost.client[0])
+    execute_cmd(multihost, "yum install -y nss-pam-ldapd")
+    execute_cmd(session_multihost, "> /etc/pam.d/sssdproxyldap")
+    execute_cmd(session_multihost, "echo 'auth        required      "
+                                   "pam_ldap.so' > /etc/pam.d/sssdproxyldap")
+    execute_cmd(session_multihost, "echo 'account     required      "
+                                   "pam_ldap.so' >> /etc/pam.d/sssdproxyldap")
+    execute_cmd(session_multihost, "echo 'password    required      "
+                                   "pam_ldap.so' >> /etc/pam.d/sssdproxyldap")
+    execute_cmd(session_multihost, "echo 'session     required      "
+                                   "pam_ldap.so' >> /etc/pam.d/sssdproxyldap")
+    domain_name = tools.get_domain_section_name()
+    domain_params = {'proxy_pam_target': 'sssdproxyldap',
+                     'id_provider': 'proxy',
+                     'proxy_lib_name': 'ldap'}
+    tools.sssd_conf('domain/' + domain_name, domain_params)
+    execute_cmd(session_multihost, "> /etc/nslcd.conf")
+    execute_cmd(session_multihost, "echo 'uid nslcd' > /etc/nslcd.conf")
+    execute_cmd(session_multihost, "echo 'gid ldap' >> /etc/nslcd.conf")
+    execute_cmd(session_multihost, f"echo "
+                                   f"'uri ldap://{session_multihost.master[0].ip}' "
+                                   f">> /etc/nslcd.conf")
+    execute_cmd(session_multihost, f"echo 'base {ds_suffix}' "
+                                   f">> /etc/nslcd.conf")
+    execute_cmd(session_multihost, "systemctl restart nslcd")
+    file_location = '/sssd/src/tests/multihost/alltests/script/sssdproxyldap.sh'
+    session_multihost.client[0].transport.put_file(os.getcwd()
+                                                   + file_location,
+                                                   '/tmp/sssdproxyldap.sh')
```

Comment:
Defining the path this make forces you to call the test runner from a specific directory. I'd suggest to check it e.g. to
```
@@ -1090,8 +1091,8 @@ def sssdproxyldap_test(session_multihost, request):
     execute_cmd(session_multihost, f"echo 'base {ds_suffix}' "
                                    f">> /etc/nslcd.conf")
     execute_cmd(session_multihost, "systemctl restart nslcd")
-    file_location = '/sssd/src/tests/multihost/alltests/script/sssdproxyldap.sh'
-    session_multihost.client[0].transport.put_file(os.getcwd()
+    file_location = '/script/sssdproxyldap.sh'
+    session_multihost.client[0].transport.put_file(os.path.dirname(os.path.abspath(__file__))
                                                    + file_location,
                                                    '/tmp/sssdproxyldap.sh')
     execute_cmd(session_multihost, f"chmod 755 /tmp/sssdproxyldap.sh")
```

Reply:
done